main.go

```diff
@@ -33,23 +31,13 @@ func startApi() {
 }
 
 func startBlockUnlocker() {
-	u := payouts.NewBlockUnlocker(&cfg.BlockUnlocker, backend, &cfg.Network)
-	u.Start()
+	// u := payouts.NewBlockUnlocker(&cfg.BlockUnlocker, backend, &cfg.Network)
```

Comment:
commentsssss.

Reply:
As addressed later the intention was to keep it (and payouts processor) for future work. But I agree that for the sake of cleanliness we can probably delete it and I'll just make an issue to keep track.

Reply:
Yep, in general we strongly prefer not to comment out code. Just delete it and if we ever want to see how old code used to look, we can look it up in the git history.

Reply:
https://github.com/dominant-strategies/go-quai-stratum/issues/9